plutus-metatheory/src/Builtin/Constant/Type.lagda.md

```diff
@@ -50,8 +50,8 @@ data TyCon : Set where
 {-# FOREIGN GHC type TypeBuiltin = SomeTypeIn DefaultUni               #-}
 {-# FOREIGN GHC pattern TyInteger    = SomeTypeIn DefaultUniInteger    #-}
 {-# FOREIGN GHC pattern TyByteString = SomeTypeIn DefaultUniByteString #-}
-{-# FOREIGN GHC pattern TyString     = SomeTypeIn DefaultUniString     #-}
-{-# FOREIGN GHC pattern TyChar       = SomeTypeIn DefaultUniChar       #-}
+{-# FOREIGN GHC pattern TyString     = SomeTypeIn DefaultUniText       #-}
+{-# FOREIGN GHC pattern TyChar       = SomeTypeIn DefaultUniText       #-}
```

Comment:
This seems suspicious, might need help from @jmchapman to figure out what to do here. Maybe just the TyString pattern?

Reply:
yes I was hoping to have James look through all these before we merge.